<?xml version="1.0"?>
<launch>
    <node pkg="tl_detector" type="tl_detector.py" name="tl_detector" output="screen" cwd="node">
    	<param name="model_path" value="$(find tl_detector)/models/udacity-sim-model/frozen_inference_graph.pb"/>
    	<param name="label_map_path" value="$(find tl_detector)/models/udacity-sim-model/label_map.pkl"/>
    	<param name="num_classes" value="4" />
    </node>
</launch>
